src/main/kotlin/listener/StarboardListener.kt

```
@@ -0,0 +1,136 @@
+    }
+    var count = 0
+    message.reactions
+        .filter {it.emoji.type== Emoji.Type.UNICODE && it.emoji.asUnicode() == Emoji.fromUnicode("⭐") }
+        .forEach {
+            count = it.count
+        }
+    if (count >= guild!!.starboardLimit) {
+        if (starboardEntry!!.starboardMessageId == 0L) {
+            var starboardMessage = try {
+                channel.retrieveMessageById(starboardEntry!!.starboardMessageId).await()
+            } catch (e: ErrorResponseException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)